src/promptflow/promptflow/_utils/load_data.py

```diff
@@ -52,13 +58,23 @@ def _pd_read_file(local_path: str, logger: logging.Logger = None) -> pd.DataFram
                 f"csv, tsv, json, jsonl, parquet. Ignoring it."
             )
             return pd.DataFrame()
+
+    # Parse the image path into absolute path for flow consumption
+    if is_json_data and enable_parse_image_path:
+        local_path = Path(local_path).resolve()
+        for index, row in df.iterrows():
+            for column_name, value in row.items():
+                df.at[index, column_name] = resolve_multimedia_data_recursively(local_path, value)
     return df
 
 
 def _bfs_dir(dir_path: List[str]) -> Tuple[List[str], List[str]]:
     """BFS traverse directory with depth 1, returns files and directories"""
     files, dirs = [], []
     for path in dir_path:
+        # Ignore the json file under .promptflow folder
+        if Path(path).name == PROMPT_FLOW_DIR_NAME:
```

Comment:
Why ignore .promptflow folder?

Reply:
There are a lots of json files under `.promptflow`.
![image](https://github.com/microsoft/promptflow/assets/111329184/e6523b92-6364-41b5-95d1-e76ae2a69b23)
If we don't ignore them and the user selects the flow folder as the input directory, it will interfere with the actual data.

Reply:
I think it is by design if the user selects the flow folder, he will read all the data.

Reply:
But `.promptflow` is generated by us. If we do not ignore it, the user must create a new folder to store the input files, otherwise an error will be raised.